### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by Gab-D-G**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -83,42 +92,69 @@ RUN curl -L --output /tmp/minc-toolkit-1.9.18.deb https://packages.bic.mni.mcgil
...
+  # integrated ANTs tools
+  ANTSPATH=${MINC_TOOLKIT}/bin
+
 #Enable ANTs
 RUN echo 'export PATH=/opt/quarantine/ANTs/bin${PATH:+:$PATH}' > /etc/profile.d/99ANTS.sh
 RUN echo 'export ANTSPATH=/opt/quarantine/ANTs/bin' >> /etc/profile.d/99ANTS.sh
+ENV PATH=/opt/quarantine/ANTs/bin${PATH:+:$PATH} \
+  ANTSPATH=/opt/quarantine/ANTs/bin
+ENV PATH=/opt/ANTs/bin${PATH:+:$PATH}
+RUN echo 'export PATH=/opt/ANTs/bin${PATH:+:$PATH}' >> /etc/profile.d/99ANTs.sh
```
Comment (gdevenyi) [inline Dockerfile:115]: This will never be run under the new entrypoint

File: Dockerfile [hunk 2]
```diff
@@ -83,42 +92,69 @@ RUN curl -L --output /tmp/minc-toolkit-1.9.18.deb https://packages.bic.mni.mcgil
 #Enable minc-toolkit
 RUN echo '. /opt/minc/1.9.18/minc-toolkit-config.sh' > /etc/profile.d/98minc.sh
```
Comment (gdevenyi) [inline Dockerfile:93]: Anything in profile.d will never run under the new entrypoint

File: Dockerfile [hunk 3]
```diff
@@ -67,6 +67,7 @@ RUN curl -L -O https://afni.nimh.nih.gov/pub/dist/bin/misc/@update.afni.binaries
     rm -f @update.afni.binaries
 
 RUN echo 'export PATH=/opt/quarantine/afni${PATH:+:$PATH}' > /etc/profile.d/99afni.sh
```
Comment (gdevenyi) [inline Dockerfile:69]: Anything in profile.d will never run under the new entrypoint

File: Dockerfile [hunk 4]
```diff
@@ -75,6 +76,14 @@ RUN curl -sSL https://raw.githubusercontent.com/nipy/nipype/master/docker/files/
     rm -rf /var/lib/apt/lists/* && \
     echo '. /etc/fsl/fsl.sh' > /etc/profile.d/99fsl.sh
```
Comment (gdevenyi) [inline Dockerfile:77]: Anything in profile.d will never run under the new entrypoint

**Review by gdevenyi — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 5]
```diff
@@ -83,42 +92,69 @@ RUN curl -L --output /tmp/minc-toolkit-1.9.18.deb https://packages.bic.mni.mcgil
...
+ENV PATH=${MINC_TOOLKIT}/bin:${MINC_TOOLKIT}/pipeline:${PATH} \
+  PERL5LIB=${MINC_TOOLKIT}/perl:${MINC_TOOLKIT}/pipeline${PERL5LIB:+:$PERL5LIB} \
+  LD_LIBRARY_PATH=${MINC_TOOLKIT}/lib:${MINC_TOOLKIT}/lib/InsightToolkit${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH} \
+  MNI_DATAPATH=${MINC_TOOLKIT}/../share:${MINC_TOOLKIT}/share \
+  MINC_FORCE_V2=1 \
+  MINC_COMPRESS=4 \
+  VOLUME_CACHE_THRESHOLD=-1 \
+  MANPATH=${MINC_TOOLKIT}/man${MANPATH:+:$MANPATH} \
+  # integrated ANTs tools
+  ANTSPATH=${MINC_TOOLKIT}/bin
```
Comment (gdevenyi) [inline Dockerfile:107]: The conflicts with the more recent ANTS install.

File: Dockerfile [hunk 6]
```diff
@@ -83,42 +92,69 @@ RUN curl -L --output /tmp/minc-toolkit-1.9.18.deb https://packages.bic.mni.mcgil
...
+  CONDA_PYTHON_EXE='${CONDA_DIR}/bin/python' \
+  # override the path with the conda environment so that it becomes the default
+  PATH=${CONDA_DIR}/bin:$PATH
 
-RUN . /etc/profile.d/99conda.sh && conda config --append channels simpleitk && \
-  conda install -y 'networkx>=2.4' 'matplotlib>=3.1.1' 'nibabel>=2.3.1' 'nilearn>=0.4.2' 'nipype>=1.1.4' 'numpy>=1.16.2' 'pandas' 'scikit-learn>=0.20.0' 'scipy' 'simpleitk>=1.2.2' 'tqdm' 'pathos' && \
+ENV RABIES=${HOME}/RABIES
+RUN mkdir $RABIES
+
+COPY rabies $RABIES/rabies
```
Comment (gdevenyi) [inline Dockerfile:139]: too many instances of the word "rabies" here in different variants

Comment (Gab-D-G) [reply Dockerfile:130]: Seems clear to me that one stands for the PATH and the other for the scripts. Do you have an alternative suggestion?

**Pull request merged**